src/snowflake/cli/plugins/nativeapp/run_processor.py

```diff
@@ -42,173 +31,6 @@ class NativeAppRunProcessor(NativeAppManager, NativeAppCommandProcessor):
-            if not self.verify_project_distribution(actual_distribution):
-                cc.warning(
-                    f"Continuing to execute `snow app run` on application package {self.package_name} with distribution '{actual_distribution}'."
-                )
-
-            # 3. If actual_distribution is external, skip comment check
-            if actual_distribution == INTERNAL_DISTRIBUTION:
-                row_comment = show_obj_row[COMMENT_COL]
-
-                if row_comment not in ALLOWED_SPECIAL_COMMENTS:
-                    raise ApplicationPackageAlreadyExistsError(self.package_name)
-
-            return
-
-        # If no application package pre-exists, create an application package, with the specified distribution in the project definition file.
-        with self.use_role(self.package_role):
-            cc.step(f"Creating new application package {self.package_name} in account.")
-            self._execute_query(
-                dedent(
-                    f"""\
-                    create application package {self.package_name}
-                        comment = {SPECIAL_COMMENT}
-                        distribution = {self.package_distribution}
-                """
-                )
-            )
-
-    def _apply_package_scripts(self) -> None:
-        """
-        Assuming the application package exists and we are using the correct role,
-        applies all package scripts in-order to the application package.
-        """
-        env = jinja2.Environment(
-            loader=jinja2.loaders.FileSystemLoader(self.project_root),
-            keep_trailing_newline=True,
-            undefined=jinja2.StrictUndefined,
-        )
-
-        queued_queries = []
-        for relpath in self.package_scripts:
-            try:
-                template = env.get_template(relpath)
-                result = template.render(dict(package_name=self.package_name))
-                queued_queries.append(result)
-
-            except jinja2.TemplateNotFound as e:
-                raise MissingPackageScriptError(e.name)
-
-            except jinja2.TemplateSyntaxError as e:
-                raise InvalidPackageScriptError(e.name, e)
-
-            except jinja2.UndefinedError as e:
-                raise InvalidPackageScriptError(relpath, e)
-
-        # once we're sure all the templates expanded correctly, execute all of them
-        try:
-            if self.package_warehouse:
-                self._execute_query(f"use warehouse {self.package_warehouse}")
-
-            for i, queries in enumerate(queued_queries):
-                cc.step(f"Applying package script: {self.package_scripts[i]}")
-                self._execute_queries(queries)
-        except ProgrammingError as err:
-            generic_sql_error_handler(
-                err, role=self.package_role, warehouse=self.package_warehouse
-            )
-
-    def _create_dev_app(self, diff: DiffResult) -> None:
```

Comment:
Sorry, where is this `_create_dev_app` being moved to? And do we need to move it at all?